src/webhooks/StaffWebhook.ts

```diff
@@ -22,6 +22,7 @@ export class StaffWebhook extends WebhookManager {
 				embed.addField(role.name, role.members.map((member) => member.toString()).join("\n") ?? "None", true);
 		});
 		embed.addField("Special Kid", "<@628829764135813160>");
+		embed.addField("Supreme Leader", "<@511209271678074891>");
```

Comment:
This can be shortened
```suggestion
embed.addFields({ name: "Special Kid", value: "<@628829764135813160>" }, { name: "Supreme Leader", value: "<@511209271678074891>"  });
```